src/components/ChattingRoom/ChatBody.tsx

```diff
@@ -0,0 +1,56 @@
+import { useRef, useEffect } from "react";
+import { useRecoilState } from "recoil";
+import styled from "styled-components";
+import chat from "../../interface/chat";
+import { chatList } from "../../store/recoil/recoil";
+import ChatBubble from "./ChatBubble";
+
+function ChatBody() {
+  const chatting = useRecoilState(chatList);
+
+  const scrollRef = useRef<HTMLDivElement>(null);
+
+  const scrollBottom = () => {
+    if (scrollRef.current) {
+      scrollRef.current.scrollTop = scrollRef.current.scrollHeight;
+    }
+  };
+
+  useEffect(() => {
+    scrollBottom();
+  }, [chatting]);
+
+  return (
+    <ChatBodyContainer ref={scrollRef}>
+      {chatting[0].map((chat: chat) => (
+        <ChatBubble
+          chatId={chat.chatId}
```

Comment:
```suggestion
      {chatting[0].map((chat: chat) => (
        <ChatBubble
          key={chat.chatId}
          chatId={chat.chatId}
```

`map` 사용하실때는 `key`값을 사용해서 고유성을 부여해주는게 좋습니다!
[참고링크](https://ko.reactjs.org/docs/lists-and-keys.html) 두고갑니다!